build.gradle

```diff
@@ -19,17 +19,6 @@ repositories {
 	}
 }
 
-// overwrite jastadd2 dependency set by jastaddgradle
-configurations {
-	all {
-		resolutionStrategy.eachDependency { DependencyResolveDetails details ->
```

Comment:
I removed that part because the new version of JastAddGradle (1.9.9) already uses JastAdd 2.1.13, so there is no need to manually override it here.